src/Tizen.Nlp/Tizen.Nlp/NlpCommon.cs

```diff
@@ -0,0 +1,175 @@
+            _msg.Connected += (sender, e) =>
+            {
+                Log.Debug(_tag, "start to register");
+                _msg.CoRegister(clientId, _noti);
+                Log.Debug(_tag, "connected callback be called");
+            };
+            Log.Debug(_tag, "start to connect");
+            _msg.Connect();
+            Log.Debug(_tag, "wait to callback of onConnected");
+        }
+
+        /// <summary>
+        /// An release session to disconnect remote tidl service.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public void Release()
+        {
+            _noti.Received -= OnReceived;
+            _msg.UnRegister();
+            _msg.Dispose();
+            _msg = null;
+
+        }
+
+        /// <summary>
+        /// Send Pos of Tag request to remote tidl service with 1 input parameters.
+        /// </summary>
+        /// <param name="info">A sentence need to be processed.</param>
+        /// <returns></returns>
+        /// <since_tizen> 5 </since_tizen>
+        public void PosTag(string info)
+        {
+            MakeRequest("pos_tag", info);
+        }
+
+        /// <summary>
+        /// Send Named Entity recognition request to remote tidl service with 1 input parameters.
+        /// </summary>
+        /// <param name="info">A sentence need to be processed.</param>
+        /// <returns></returns>
+        /// <since_tizen> 5 </since_tizen>
+        public void NeChunk(string info)
+        {
+            MakeRequest("ne_chunk", info);
+        }
+
+        /// <summary>
+        /// Send language detect request to remote tidl service with 1 input parameters.
+        /// </summary>
+        /// <param name="info">A sentence need to be processed.</param>
+        /// <returns></returns>
+        /// <since_tizen> 5 </since_tizen>
+        public void LangDetect(string info)
+        {
+            MakeRequest("langdetect", info);
+        }
+
+        /// <summary>
+        /// Send Lemmatize request to remote tidl service with 1 input parameters.
+        /// </summary>
+        /// <param name="info">A sentence need to be processed.</param>
+        /// <returns></returns>
+        /// <since_tizen> 5 </since_tizen>
+        public void Lemmatize(string info)
+        {
+            MakeRequest("lemmatize", info);
+        }
+
+        /// <summary>
+        /// Send word tokenize request to remote tidl service with 1 input parameters.
+        /// </summary>
+        /// <param name="info">A sentence need to be processed.</param>
+        /// <returns></returns>
+        /// <since_tizen> 5 </since_tizen>
+        public void WordTokenize(string info)
+        {
+            MakeRequest("word_tokenize", info);
+        }
+
+    }
+
+    /// <summary>
+    /// This custom class extend from EventArgs to obtain Bundle object.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public class CustomEventArg : EventArgs
```

Comment:
Please move this class to CustomEventArg.cs file. I strongly recommend that a single .cs file has only one public class.
And the name of EventArgs should indicate what kind of Event it is related to. How about change to MessageReceivedEventArgs.